tests/output/ConsoleInteractionTest.py

```diff
@@ -869,12 +869,22 @@ class ShowBearsTest(unittest.TestCase):
     def setUp(self):
         self.console_printer = ConsolePrinter(print_colored=False)
 
+    # This list should be static
```

Comment:
static?

Reply:
@Makman2 it should not be a data member of the unittest class

Reply:
but it is one now, isn't it? ;)

Reply:
@Makman2 it basically means that the list shouldn't be stored and accessed with the self parameter.
For some reason self.list cannot be accessed in the check logs decorator

Reply:
Yeah because it's has class-scope. `self` is not accesible in the class, but just in bound methods ;) I think this is self-explanatory, but I understand your point. Though you should refrain at least from words like "static", the concept of "static" like in C or similar languages does not really apply in Python :)

Reply:
Then should I remove the comment or modify it for it to fit the current scenario better ?